lib/utils/utils.go

```diff
@@ -471,7 +475,11 @@ func ReadOrMakeHostUUID(dataDir string) (string, error) {
 	if !trace.IsNotFound(err) {
 		return "", trace.Wrap(err)
 	}
-	id = uuid.New().String()
+	rawID, err := uuid.NewRandom()
```

Comment:
Do you know if it's possible that this returns an empty UUID with a nil error?

Reply:
It is not possible. `uuid.UUID` is an alias for `[16]byte`, so the emptiest it can get is a string of zeros (and `NewRandom()` ensures that there are a few nonzero bytes).